samples/python/agents/semantickernel/agent.py

```diff
@@ -93,18 +93,53 @@ class SemanticKernelTravelAgent:
     SUPPORTED_CONTENT_TYPES = ['text', 'text/plain']
 
     def __init__(self):
-        api_key = os.getenv('OPENAI_API_KEY', None)
-        if not api_key:
-            raise ValueError('OPENAI_API_KEY environment variable not set.')
-
-        model_id = os.getenv('OPENAI_CHAT_MODEL_ID', 'gpt-4.1')
+        # Check if Azure OpenAI should be used
+        use_azure = os.getenv('ENABLE_AZURE_OPENAI', 'false').lower() == 'true'
```

Comment:
From a sample perspective, I'd prefer to keep this as simple as possible. We do plan to introduce other agent types here at some point (like the `AzureAIAgent`). This constructor is going to grow and become hard to manage.

Once we do `load_dotenv()` the env vars will be available when we create the `AzureChatCompletion()` or `OpenAIChatCompletion()` objects. It's less code to show:

`chat_service = AzureChatCompletion()` then to override everything with kwargs. Pydantic settings will handle the env vars underneath.